testsuite/tests/badly-ordered-deps/aux.ml

```diff
@@ -0,0 +1 @@
+let value = Main.value
```

Comment:
Strange, but true - in 2023, DOS compatibility still means files named `aux.*` aren't allowed in Windows! This file just needs to be renamed to something else.